frontends/web/src/components/aopp/vasp.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { h, JSX, RenderableProps } from 'preact';
+import * as styles from './vasp.css';
+import TwentyOneAnalyticsLogo from '../../assets/exchanges/logos/21analytics.png';
+import BitcoinSuisseLogo from '../../assets/exchanges/logos/bitcoin_suisse.png';
+import BittrLogo from '../../assets/exchanges/logos/bittr.png';
+import BityLogo from '../../assets/exchanges/logos/bity.png';
+import PocketBitcoinLogo from '../../assets/exchanges/logos/pocketbitcoin.svg';
+
+interface VASPProps {
+    fallback?: JSX.Element;
+    hostname: string;
+    prominent?: boolean;
```

Comment:
remove this? i only see one use of this component and it is always prominent.

Reply:
Please look again, only on the first view (user-confirmation) it is prominent, see here https://github.com/digitalbitbox/bitbox-wallet-app/pull/1405/files#diff-c4c5f0fa92cdc4f4a6fbdd048df4be6bea47bea9db5204b4656e7ac59ba96869R122

but all other views it is used without `prominent`